examples/Atlas/Atlas.m

```diff
@@ -79,6 +82,13 @@
         end
       end
 
+      % Add a force on a specified link if we want!
+      if ~isempty(options.external_force)
+        obj.external_force = 1;
```

Comment:
since this is the linkID where the force is applied (but just looks like boolean true), could you add a comment saying that you're hard-coding it to be the pelvis here?